Makefile

```diff
@@ -909,7 +909,7 @@ endif
 # Utils                                                                       #
 ###############################################################################
 
-.PHONY: lint gjslint hint csslint
+.PHONY: lint gjslint hint csslint eslint
```

Comment:
can you also add some text in the Makefile header ? there is a comment section about linting files.